### ISSUE TITLE
Issue with decimals when withdrawing

### ISSUE THREAD

**Issue by Sukender** [swashapp/swash-ui#11]
When withdrawing, under version 8b575d61 (1.0.8+) with Chrome desktop and Firefox desktop, I have a blank white modal window after clicking "Confirm and send". Browser's console says:

```
browser-polyfill.js:1126 Uncaught (in promise) Error: invalid BigNumber string value (arg="value", value="0.328", version=4.0.37)
    at wrappedSendMessageCallback (browser-polyfill.js:1126)
```

I suspect that the "0.328" value I entered is not parsed as a number, because I'm on a system where `.` is **not** the decimal separator. Indeed, system and browsers' locale is FRA, and decimal separator is `,`, and thousand separator is ` ` (space). I tried to enter "0,328" (comma instead of dot) but I then had an "Amount value is not valid" message.

To sum up:
1. The exception should be caught and a message should be displayed.
2. The parsing should handle this. My very personal suggestion is to: 
   - First try to parse using current locale.
   - Then try with the "C" locale (= default) if the first fails.
   - And use the same parsing code in all places.

**Comment by Sukender** [comment]
Update : in 572320bc (1.0.8+), I know have a message in the white modal, which says "Reason: Insufficient ETH to pay for the gas fee".

Thanks for fixing the parsing issue. But what is the minimum threshold to send? Must I first send ETh to this wallet to allow transferring!?
